pkgs/games/vvvvvv/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchurl, fetchFromGitHub, requireFile
+, SDL2, SDL2_mixer, cmake, ninja
+, fullGame ? false }:
+
+let
+  dataZip = if fullGame then requireFile {
+    name = "data.zip";
+    sha256 = "1q2pzscrglmwfgdl8yj300wymwskh51iq66l4xcd0qk0q3g3rbkg";
+    message = ''
+      In order to install VVVVVV, you must first download the game's
+      data file (data.zip) as it is not released freely.
+      Once you have downloaded the file, place it in your current
+      directory, use the following command and re-run the installation:
+      nix-prefetch-url file://\$PWD/data.zip
+    '';
+  } else fetchurl {
+    url = https://thelettervsixtim.es/makeandplay/data.zip;
```

Comment:
This is redistributable?

Reply:
There's a general exception to the license that this is distributable with the "Make and Play Edition" on Linux distributions: https://github.com/TerryCavanagh/VVVVVV/issues/20#issuecomment-577424126